dashboard/app/models/concerns/user/verifiable.rb

```diff
@@ -0,0 +1,29 @@
+module User::Verifiable
```

Comment:
Should we also move this to the Teacher model?
```suggestion
module Teacher::Verifiable
```

Reply:
This is a good point.
Since students can't be verified I would make sense to just create the concern for teachers.

Reply:
This is definitely the north star. However making only this change breaks existing functionality and tests were these methods are being called on student users. Because this would only be included in the Teacher class, if you call any of these methods on a Student, these methods are undefined. My preference would be to merge this PR first, which introduces these methods being moved to the concern. From here, we can move where the concern is included, and update places in the code where these methods are being called on students.